accelerometer/summariseEpoch.py

```diff
@@ -13,13 +13,13 @@
 from datetime import timedelta
 
 def getActivitySummary(epochFile, nonWearFile, summary,
-    activityClassification=True, timeZone='Europe/London', 
+    activityClassification=True, timeZone='Europe/London',
     startTime=None, endTime=None,
-    epochPeriod=30, stationaryStd=13, minNonWearDuration=60, 
-    mgMVPA=100, mgVPA=425, 
+    epochPeriod=30, stationaryStd=13, minNonWearDuration=60,
+    mgMVPA=100, mgVPA=425,
     activityModel="activityModels/doherty-may20.tar",
     intensityDistribution=False, useRecommendedImputation=True,
-    psd=False, fourierFrequency=False, fourierWithAcc=False, m10l5=False, 
+    psd=False, fourierFrequency=False, fourierWithAcc=False, m10l5=False,
     verbose=False):
     """Calculate overall activity summary from <epochFile> data
```

Comment:
Nothing to see here. This is just a quirk of my editor (VSCode) that automatically removes trailing spaces (a best practice) in the file.

Reply:
No problem, at some point I will apply a linter to the project